Gems/Multiplayer/Code/Source/Components/NetworkCharacterComponent.cpp

```diff
@@ -156,19 +156,6 @@ namespace Multiplayer
         }
     }
 
-    bool NetworkCharacterComponent::IsOnGround() const
```

Comment:
Why is this being removed?

Reply:
From what I could tell, this component never connected to CharacterGameplayRequestBus so this logic wasn't being used. The options were either connect the bus or remove the logic so I opted to remove it for now.